src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -694,4 +694,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_UnableToSignPackage" xml:space="preserve">
     <value>Unable to sign package.</value>
   </data>
+  <data name="Warning_FileExcludedByDefault" xml:space="preserve">
+    <value>File '{0}' was not added to the package. Files and folders starting with '.' or ending with '.nupkg' are excluded by default. To include this file, use -NoDefaultExcludes from the commandline</value>
```

Comment:
This will only hit for nuspec pack right?
Like the -NoDefaultExcludes makes sense in nuspec pack?

Reply:
No this applies to all pack scenarios

Reply:
What about MSBuild task? How do you disable NoDefaultExcludes in CSPROJ?

Reply:
@pakrym you are right. that doesn't exist right now, and I just realized you filed a bug for it. I'll see if I can get this enabled for 15.7 

https://github.com/NuGet/Home/issues/6450